Type error in v14.0

**Describe the bug**
```bash
../../node_modules/.pnpm/twemoji@14.0.0/node_modules/twemoji/index.d.ts:32:1 - error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.

32 const twemoji: {
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to https://github.com/Saul-Mirone/milkdown
2. `git checkout renovate/all`
3. `pnpm install && pnpm build --filter=@milkdown/plugin-emoji`
4. See error

**Expected behavior**
Typescript compile can pass with no error.

**Screenshots**
If applicable, add screenshots to help explain your problem.
![image](https://user-images.githubusercontent.com/10047788/158066232-69aab94b-a393-4f02-81d8-552962cca10f.png)

**Environment**
For web, specify your OS and browser version.  For mobile, specify device, OS
and version.  For libraries, what version of build tools are you using?

**Additional context**
Add any other context about the problem here.

--- a/index.d.ts
+++ b/index.d.ts
@@ -29,12 +29,15 @@ declare interface TwemojiOptions {
   attributes?(): void;
 }
 
-declare const twemoji: {
+declare type twemoji = {
   convert: {
     fromCodePoint(hexCodePoint: string): string;
     toCodePoint(utf16surrogatePairs: string): string;
   };
-  parse(node: HTMLElement | string, options?: TwemojiOptions): string;
+  parse(node: HTMLElement | string, options?: TwemojiOptions): void;
 };
 
-export default twemoji;
+declare module "twemoji" {
+  const twemoji: twemoji;
+  export default twemoji;
+}

--- a/index.d.ts
+++ b/index.d.ts
@@ -29,7 +29,7 @@ declare interface TwemojiOptions {
   attributes?(): void;
 }
 
-export const twemoji: {
+declare const twemoji: {
   convert: {
     fromCodePoint(hexCodePoint: string): string;
     toCodePoint(utf16surrogatePairs: string): string;

--- a/index.d.ts
+++ b/index.d.ts
@@ -29,7 +29,7 @@ declare interface TwemojiOptions {
   attributes?(): void;
 }
 
-declare type twemoji = {
+declare type Twemoji = {
   convert: {
     fromCodePoint(hexCodePoint: string): string;
     toCodePoint(utf16surrogatePairs: string): string;
@@ -37,7 +37,7 @@ declare type twemoji = {
   parse(node: HTMLElement | string, options?: TwemojiOptions): void;
 };
 
-declare module "twemoji" {
-  const twemoji: twemoji;
+declare module 'twemoji' {
+  const twemoji: Twemoji;
   export default twemoji;
 }

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,4 +1,4 @@
-interface TwemojiOptions {
+declare interface TwemojiOptions {
   /**
    * Default: MaxCDN
    */
@@ -29,13 +29,12 @@ interface TwemojiOptions {
   attributes?(): void;
 }
 
-const twemoji: {
+export const twemoji: {
   convert: {
     fromCodePoint(hexCodePoint: string): string;
     toCodePoint(utf16surrogatePairs: string): string;
   };
-  parse(node: HTMLElement | string, options?: TwemojiOptions): void;
+  parse(node: HTMLElement | string, options?: TwemojiOptions): string;
 };
 
 export default twemoji;
-

